src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  position: relative;
+  @extend %flex-view--center;
+  @extend %font;
+  @extend %calendar-width;
+  margin: auto;
```

Comment:
Don't add margin and position to a block, only for element. Use mix.
 More details in video.
[video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911)
![Screenshot 2022-07-22 at 12 30 04](https://user-images.githubusercontent.com/17098736/180409945-95153649-8903-4a15-bc07-63f2edef30de.png)